test/index.js

```diff
@@ -2134,25 +2220,21 @@ describe('wait()', () => {
 
     it('delays for timeout ms', async () => {
 
-        // DO NOT USE Hoek.Bench() to time this, as its test depends on Hoek.wait()
```

Comment:
This still seems like a useful comment to me, but not a huge deal either way.

Reply:
Yeah, I removed it since the updated test does not use timers.